Remote modify: Added kerb_ticket option for HDFS (#2387)
iterative/dvc.org: content/docs/command-reference/remote/modify.md

> The user name may contain sensitive user info. Therefore, it's safer to add it
> with the `--local` option, so it's written to a Git-ignored config file.

> - `kerb_ticket` - path to Kerberos ticket cache to connect to Kerberos-secured HDFS clusters.

Reviewer: Should we link "Kerberos ticket cache" somewhere or is it something HDFS/Kerb users will definitely know already?

Possible links:
- https://arrow.apache.org/docs/python/generated/pyarrow.fs.HadoopFileSystem.html#pyarrow.fs.HadoopFileSystem
- https://web.mit.edu/kerberos/krb5-1.12/doc/basic/ccache_def.html
- https://www.hvr-software.com/docs/5/location-class-requirements/requirements-for-hdfs/hdfs-authentication-and-kerberos#HDFSAuthenticationandKerberos-AccessingKerberizedClusterswithTicketCacheFile

> - `kerb_ticket` - path to Kerberos ticket cache to connect to Kerberos-secured HDFS clusters.
> 
>   ```dvc
>   $ dvc remote modify myremote kerb_ticket /path/to/kerb_ticket

Reviewer: ````suggestion
- `kerb_ticket` - path to the Kerberos ticket cache for Kerberos-secured HDFS
  clusters

  ```dvc
  $ dvc remote modify --local myremote \
                              kerb_ticket /path/to/ticket/cache
````
